src/untangled/server/impl/components/access_token_handler.clj

```diff
@@ -96,9 +96,8 @@
               token (get-token request)]
           (if-not (valid-token? token merged-options)
             (let [{:keys [invalid-token-handler]} merged-options]
-              (if-let [_ok-anyway? ((or invalid-token-handler (constantly false)) request)]
-                (handler request)
-                {:status 401}))
+              ((or invalid-token-handler (constantly false)) request)
+              (handler request))
```

Comment:
This doesn't seem right, wouldn't you always be returning false if there's no `invalid-token-handler`?
Also wouldn't it not even use that return value, but always call `(handler request)`?